src/contracts/GPv2AccessControl.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: LGPL-3.0-or-newer
+pragma solidity ^0.6.12;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/EnumerableSet.sol";
+
+/// @title Gnosis Protocol v2 Access Control Contract
+/// @author Gnosis Developers
+contract GPv2AccessControl is Ownable {
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    EnumerableSet.AddressSet private solvers;
+
+    function addSolver(address solverAddress) public {
+        solvers.add(solverAddress);
```

Comment:
I'm surprised solidity can auto-convert bytes32 from/to address.

Reply:
Yea although I noticed that the hex value 0x00 is an acceptable default (as seen in the base contract for AccessControl) but that one can not simply use 0x01 to allocate another role value.